app/src/main/java/com/sdp/movemeet/HomeUnregister.java

```diff
@@ -0,0 +1,38 @@
+package com.sdp.movemeet;
```

Comment:
It is a detail, but shouldn't this class be called "MainActivityUnregistered", or something like that ? From my understanding, this isn't the home, but rather the main activity but without the user  menu

Reply:
I can rename the class !